app/components/SidebarLayout.tsx

```diff
@@ -110,7 +110,7 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
               <SideBarListItem
                 isCollapsed={isCollapsed}
                 isCurrentPage={segments.includes("locations")}
-                Icon={FaMap}
+                Icon={FaLocationDot}
```

Comment:
i changed the sidebar icon to the map marker, this feels a lot more intuitive to me than the folding map icon